src/internal/log/log.go

```diff
@@ -191,13 +191,17 @@ func (l *logger) LogAtLevelFromDepth(request interface{}, response interface{},
 		fields["response"] = response
 	}
 	if err != nil {
-		// "err" itself might be a code or even an empty struct
-		fields["error"] = err.Error()
-		var frames []string
-		errors.ForEachStackFrame(err, func(frame errors.Frame) {
-			frames = append(frames, fmt.Sprintf("%+v", frame))
-		})
-		fields["stack"] = frames
+		if goerr, ok := err.(error); ok {
```

Comment:
I don't think we should define our own type for this check, after looking this over.  We should check to see if it is a grpc status, or implements a `GRPCStatus() *status.Status` method.  If it doesn't, log it.  If it does, switch on the Code (which is like an HTTP status code), and don't log errors originating on the client.  By default we should log errors, we should only skip logging for specific codes.

Then when we want to stop an error from making it into the logs, we implement GRPCStatus for that error type.  That has the additional benefit of being able to inspect the errors using the grpc libraries.